tests/attacks/test_newtonfool.py

```diff
@@ -21,16 +21,21 @@
 import unittest
 
 import numpy as np
+from tests.attacks.utils import backend_test_classifier_type_check_fail
+from tests.utils import (
+    TestBase,
+    get_image_classifier_kr,
+    get_image_classifier_pt,
+    get_image_classifier_tf,
+    get_tabular_classifier_kr,
+    get_tabular_classifier_pt,
+    get_tabular_classifier_tf,
+)
```

Comment:
I think we should keep the imports from `tests` after the imports from `art` because they are not third-party and to keep it consistent across all tests.